sample/github/README.md

```diff
@@ -6,156 +6,156 @@ deployed to receive the webhook's event deliveries and forward them into a
-4. Install a ClusterBus
-
-update `knative/eventing/config/buses/stub-bus.yaml`, changing `kind` to `ClusterBus`, like:
-
-```yaml
-apiVersion: channels.knative.dev/v1alpha1
-kind: ClusterBus
-metadata:
-  name: stub
-spec:
-  dispatcher:
-    name: dispatcher
-    image: github.com/knative/eventing/pkg/buses/stub
-    args: [
-      "-logtostderr",
-      "-stderrthreshold", "INFO",
-    ]
-``` 
-
-and apply the stub bus
-
-```shell
-ko apply -f config/buses/stub
-```
-
-5. Install GitHub as an event source
-
-```shell
-ko apply -f pkg/sources/github/
-```
-
-5. Check that the GitHub is now showing up as an event source and there's an event type for *pullrequests*
-
-```shell
-kubectl get eventsources
-kubectl get eventtypes
-```
-
-6. Create a [personal access token](https://github.com/settings/tokens) to GitHub repo that you can use to register
-   webhooks with the GitHub API. Also decide on a token that your code will use to authenticate the incoming webhooks
-   from GitHub (*accessToken*). Update `sample/github/githubsecret.yaml` with those values. If your generated access
-   token is `'asdfasfdsaf'` and you choose your *secretToken* as `'password'`, you'd modify
-   `sample/github/githubsecret.yaml` like so:
-
-```yaml
-apiVersion: v1
-kind: Secret
-metadata:
-  name: githubsecret
-type: Opaque
-stringData:
-  githubCredentials: >
-    {
-      "accessToken": "asdfasfdsaf",
-      "secretToken": "password"
-    }
-```
-
-## Running
-
-In response to a pull request event, the _legit_ Service will add `(looks pretty legit)` to the PR title.
-
-Deploy the _legit_ service via:
-
-```shell
-ko apply sample/github/legit-service.yaml
-```
+In response to a pull request event, the sample app _legit_ Service will add 
+`(looks pretty legit)` to the PR title.
 
-Once deployed, you can inspect the created resources with `kubectl` commands:
-
-```shell
-# This will show the Service that we created:
-kubectl get service.serving.knative.dev -o yaml
-
-# This will show the Route that the Service created:
-kubectl get route -o yaml
-
-# This will show the Configuration that the Service created:
-kubectl get configurations -o yaml
+## Prerequisites
 
-# This will show the Revision that was created by the Configuration:
-kubectl get revisions -o yaml
-```
+- A Kubernetes cluster with Knative installed. Follow the
+  [installation instructions](https://github.com/knative/docs/blob/master/install/README.md)
+  if you need to create one.
+- [Docker](https://www.docker.com/) installed and running on your local machine,
+  and a Docker Hub account configured (you'll use it for a container registry).
+- The core Knative eventing tools installed. You can install them with:
+  ```shell
+  kubectl apply -f https://storage.googleapis.com/knative-releases/eventing/latest/release.yaml
+  ```
+- A custom domain is setup to allow for GitHub to be able to call into the cluster:
```

Comment:
The grammar here doesn't line up with the other bullets. I'd rephrase this to `A domain name that allows GitHub to call into the cluster`.